TP-Sampler/src/view.js

```diff
@@ -715,7 +725,7 @@ View.prototype = {
         // variable name label
         var variableLabelClipping = s.path(slice.path);
         variableLabel = this.createSpinnerLabel(i, slice.center.x, slice.center.y, varTextSize,
-        variableLabelClipping, Math.max(1, 10 - variables.length));
+                                                  variableLabelClipping, Math.max(10, 10 - variables.length));
```

Comment:
Looks like maybe a bunch of tabs were inserted?

Reply:
Just wanted to line it up as part of the variables passed into the `createSpinnerLabel`.